LocalSettings.php

```diff
@@ -2381,23 +2379,16 @@
 			'managewiki-editdefault',
 			'mwoauthmanageconsumer',
 			'mwoauthmanagemygrants',
-			'mwoauthproposeconsumer',
 			'mwoauthsuppress',
-			'mwoauthupdateownconsumer',
 			'mwoauthviewprivate',
 			'mwoauthviewsuppressed',
-			'read',
```

Comment:
it should not be possible to revoke read from * except by making the wiki private.